entrypoint.sh

```diff
@@ -84,29 +85,36 @@ for i in "${!REPOS[@]}"; do
   REPOLIST="${REPOLIST} --repofrompath=repo${i},${REPOS[$i]}"
 done
 
+YUMDOWNLOADER="yumdownloader --archlist=x86_64 --archlist=noarch --disablerepo=* --releasever=${VERSION_ID} ${REPOLIST}"
+
+# Install CRI-O / hyperkube / oc RPMs updating RPM DB
+mkdir /tmp/working
+pushd /tmp/working
+  # Extract RPM DB
+  mkdir -p usr/lib
+  ostree --repo=/srv/repo checkout "${REF}" --subpath /usr/lib/rpm --user-mode $(pwd)/usr/lib/rpm
+
+  # Download crio from modular repo
+  sed -i 's/enabled=0/enabled=1/g' /etc/yum.repos.d/fedora-updates-testing-modular.repo
+  dnf --installroot=$(pwd) --releasever=${VERSION_ID} module enable -y cri-o:${CRIO_VERSION}
+  ${YUMDOWNLOADER} --destdir=/tmp/rpms --enablerepo=updates-testing-modular cri-o cri-tools
+  # Install additional RPMs
+  rpm -ivh ${ADDON_RPMS}/* --nodeps --dbpath $(pwd)/usr/lib/rpm --prefix $(pwd)
```

Comment:
I wonder whether we can't use dnf itself to install the rpms?
Also maybe instead of `yumdownloader`, `dnf --downloadonly` could be used?

Reply:
Hmm, yeah, I'll give it a try